### PR TITLE
Updated the auto configuration to respect a user-specified database name

### PR DESCRIPTION
**Pull request by Ceiu**
- Using the auto configuration with a custom database name will now
  properly generate configuration with the specified DB name

### CODE REVIEW

**Comment by awood** [comment]
Buildbot, retest this please

**Comment by awood** [comment]
Looks like this needs to get rebased to master to pick up some spec test fixes

**Comment by Ceiu** [comment]
Ahh, yeah. Rebased, hopefully that takes care of it.

File: server/bin/deploy [hunk 1]
```diff
@@ -108,7 +108,7 @@ create_var_cache_candlepin() {
 }
 
 autoconf() {
-  buildr -s -e $1 erb $2 $3
+  buildr -s -e $1 app_db_name=$APP_DB_NAME erb $2 $3
```
Comment (awood) [inline server/bin/deploy]: Where is `APP_DB_NAME` set to a default?

Comment (Ceiu) [inline server/bin/deploy]: At the top (line 3). Same applies to the Gutterball script.

Comment (awood) [inline server/bin/deploy]: > At the top (line 3). Same applies to the Gutterball script.

Ah!  Didn't realize it was a pre-existing variable.

**Pull request merged**